src/engine-actions/translate-swap.js

```diff
@@ -5,6 +5,13 @@ const {
   DEFAULT_MIN_FINAL_DELTA
 } = CLTV_DELTA
 
+/**
+ * Default fee limit for swap routes.
+ * We expect to route swaps through a fee-less hub.
+ * @todo Make this value dynamic.
+ */
+const DEFAULT_FEE_LIMIT = 0
```

Comment:
@treygriffith do we want to throw this in `config.json`?

Reply:
I don't think so, because I expect that when we make this dynamic it will be by adding a parameter to the call rather than configuring it at startup. But I'm open to pushback.

Reply:
Im fine with this